.golangci.yml

```diff
@@ -11,12 +11,22 @@ linters:
     - vet
     - unused
     - misspell
+    - gosec
   disable:
     - errcheck
 
 issues:
```

Comment:
- How about putting them in the gosec section? We don't have to rely on the textual representation of the errors. https://golangci-lint.run/usage/linters/#gosec
- Are they all irrelevant for this project, or should be addressed later? It would be better to add some context.

Reply:
Good point. I made suggested changes in the new version.

As to the excluded issues, they are not related to `G601: Implicit memory aliasing in for loop`.

Can we cut new tickets to address them separately one by one? as there are many of them...